packages/driver/src/cy/net-stubbing/events/response.ts

```diff
@@ -21,7 +21,7 @@ export const onResponse: HandlerFn<CyHttpMessages.IncomingResponse> = async (Cyp
   const { routeId } = subscription
   const request = getRequest(routeId, frame.requestId)
 
-  parseJsonBody(res)
+  const bodyParsed = parseJsonBody(res)
```

Comment:
We need to remember whether the body was parsed because the intercept could potentially remove the "content-type: application/json" header which would result in the body being parsed but not stringified (if `stringifyJsonBody` would rely on this header as in my initial implementation). This was exhibited by the "can delete a response header" test in net-stubbing.